The configure stuff is a inconsequent:
- even if neither alsa nor gstreamer support is enabled, SBC_LIBS gets substituted by libsbc.la
  which doesn't get build without alsa or gstreamer. Making this conditional helps.
- ipctest needs both libipc.la and libsbc.la and fails if SBC_LIBS/SBC_CFLAGS are empty,
  making the build conditional helps again.
--- acinclude.m4.orig	2009-04-06 16:26:14.570780241 +0200
+++ acinclude.m4	2009-04-06 16:26:59.540779148 +0200
@@ -330,8 +330,10 @@
 	AC_SUBST([GDBUS_CFLAGS], ['-I$(top_srcdir)/gdbus'])
 	AC_SUBST([GDBUS_LIBS], ['$(top_builddir)/gdbus/libgdbus.la'])
 
-	AC_SUBST([SBC_CFLAGS], ['-I$(top_srcdir)/sbc'])
-	AC_SUBST([SBC_LIBS], ['$(top_builddir)/sbc/libsbc.la'])
+	if (test "${alsa_enable}" = "yes" || test "${gstreamer_enable}" = "yes"); then
+		AC_SUBST([SBC_CFLAGS], ['-I$(top_srcdir)/sbc'])
+		AC_SUBST([SBC_LIBS], ['$(top_builddir)/sbc/libsbc.la'])
+	fi
 
 	AM_CONDITIONAL(SNDFILE, test "${sndfile_enable}" = "yes" && test "${sndfile_found}" = "yes")
 	AM_CONDITIONAL(NETLINK, test "${netlink_enable}" = "yes" && test "${netlink_found}" = "yes")
--- audio/Makefile.am.orig	2009-04-06 16:47:21.240681272 +0200
+++ audio/Makefile.am	2009-04-06 17:42:27.240597715 +0200
@@ -60,10 +60,16 @@
 
 libipc_la_SOURCES = ipc.h ipc.c
 
+if AUDIOPLUGIN
+if SBC
+
 noinst_PROGRAMS = ipctest
 
 ipctest_LDADD= libipc.la @SBC_LIBS@ @GLIB_LIBS@
 
+endif
+endif
+
 AM_CFLAGS = -fvisibility=hidden @SBC_CFLAGS@ \
 		@BLUEZ_CFLAGS@ @DBUS_CFLAGS@ @GLIB_CFLAGS@ @GDBUS_CFLAGS@
 
